src/bootstrap/compile.rs

```diff
@@ -216,6 +216,11 @@ impl Step for StdLink {
         let libdir = builder.sysroot_libdir(target_compiler, target);
         add_to_sysroot(&libdir, &libstd_stamp(build, compiler, target));
 
+        if target.contains("windows") {
+            copy(&build.src.join("src/etc/longPathAware.manifest"),
+                 &libdir.join("longPathAware.manifest"));
```

Comment:
Is copying the manifest around really the best idea? Couldn't we just point the linker to where the manifest is originally instead of copying it around?

Reply:
There's multiple copies of the compiler during bootstrap and each get their own manifest file.

Reply:
Ah right, you're copying during bootstrap, not when rustc is compiling normal code. Disregard me.